internal/controllers/clusterrequest_controller.go

```diff
@@ -436,7 +435,8 @@ func (t *clusterRequestReconcilerTask) renderClusterInstanceTemplate(
 		"Cluster": t.clusterInput.clusterInstanceData,
 	}
 
-	renderedClusterInstance, err := utils.RenderTemplateForK8sCR(
+	renderedClusterInstance := &siteconfig.ClusterInstance{}
+	renderedClusterInstanceUntyped, err := utils.RenderTemplateForK8sCR(
```

Comment:
Could we consider renaming `renderedClusterInstanceUntyped` to `renderedClusterInstanceUnstruct`? I think it conveys its purpose a bit better.